### PR TITLE
refactor: split preview sources for clarity

### PR DESCRIPTION
**Pull request by tbouffard**
Contains tricks in the gulp build
- change the site url with an environment variable. We can parametrize the site url for PR preview (and later live preview of the master branch of this repository). Previously, it was always set to http://localhost:5252/
- ensure that the navigation is always opened to currently displayed page

covers https://github.com/bonitasoft/bonita-documentation-site/issues/325

### Information of the PR initialization

Cherry-pick work started in #85 (force-pushed the feat/troubleshooting_synthax branch from 75f361d to 27db6bb)
  - history: https://github.com/bonitasoft/bonita-documentation-theme/commits/75f361de422f94487262a6419ce3727b6226b208
  - cherry picked commit with adaptation: https://github.com/bonitasoft/bonita-documentation-theme/commit/c2699bb74786918fb2588a4d39dec2a0ac7a2223

### CODE REVIEW

File: preview-src/admonitions.adoc [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+|===
+++++
++
+. blabla
+----
+====
+
+
+
+
```
Comment (jeromecambon) [inline preview-src/admonitions.adoc:306]: Not sure we need these examples below

File: preview-src/admonitions.adoc [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+.Click to see the *source code*
+[%collapsible]
+====
+[source,asciidoc]
+----
+[NOTE.bonitacloud,caption=Bonita cloud]
+++++
+*Lorem Ipsum* is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. `And here is some code!`. xref:nothing[And a Link!].
+++++
+
```
Comment (jeromecambon) [inline preview-src/admonitions.adoc:125]: Not usre we need these "Raw admonitions" examples below

File: preview-src/troubleshooting.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
+= How to write a clean troubleshooting section
```
Comment (jeromecambon) [inline preview-src/troubleshooting.adoc:1]: Is the content is on purpose? To me, the Troubleshooting exampes from misc.adoc are better...

**Review by benjaminParisel — APPROVED** [review]

**Review by jeromecambon — APPROVED** [review]
Seems good!

**Pull request merged**